src/Plugin/tmgmt/Translator/ThebigwordTranslator.php

```diff
@@ -0,0 +1,872 @@
+        $mapping = reset($mappings);
+        $ids = $mapping->getRemoteData('files');
+        $ids = array_keys($ids);
+        $file_id = reset($ids);
+        if (isset($files[$file_id])) {
+          try {
+            $this->addFileDataToJob($job, $state, $file_id);
+          }
+          catch (TMGMTException $e) {
+            $this->sendFileError('RestartPoint01', $file_id, $e->getMessage(), NULL, $job_item);
+            $job->addMessage('Error fetching the job item: @job_item.', array('@job_item' => $job_item->label()), 'error');
+            $had_errors = TRUE;
+            $not_translated++;
+            continue;
+          }
+          $translated++;
+        }
+        else {
+          $not_translated++;
+        }
+      }
+    }
+    catch (TMGMTException $e) {
+      drupal_set_message('Could not pull translation resources.', array(), 'error');
+      return FALSE;
+    }
+    finally {
+      if ($had_errors) {
+        $this->confirmUpload($project_id, 'RestartPoint01');
+      }
+      return [
+        'translated' => $translated,
+        'untranslated' => $not_translated,
+      ];
+    }
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function requestJobItemsTranslation(array $job_items) {
+    /** @var JobItemInterface $job_item */
+    foreach ($job_items as $job_item) {
+      $this->setTranslator($job_item->getJob()->getTranslator());
+
+      try {
+        $job_item_id = $job_item->id();
+        $project_id = $this->newTranslationProject($job_item_id, $job_item_id, $job_item->getJob()->getSetting('required_by'), $job_item->getJob()->getSetting('quote_required'), $job_item->getJob()->getSetting('category'));
+
+        /** @var RemoteMapping $remote_mapping */
+        $remote_mapping = \Drupal::entityTypeManager()
+          ->getStorage('tmgmt_remote')
+          ->create([
+            'tjid' => $job_item->getJob()->id(),
+            'tjiid' => $job_item->id(),
+            'data_item_key' => 'tmgmt_thebigword',
+            'remote_identifier_1' => 'ProjectId',
+            'remote_identifier_2' => $project_id,
+            'files' => [],
+          ]);
+        $remote_mapping->save();
+        $this->sendFiles($job_item);
+        // Confirm is required to trigger the translation.
+        $confirmed = $this->confirmUpload($project_id, 'ReferenceAdd');
+        if ($confirmed != 1) {
+          $message = 'Not all the references had been confirmed.';
+          $this->sendFileError('RestartPoint01', $message, NULL, $job_item);
+          throw new TMGMTException($message);
+        }
+        $confirmed = $this->confirmUpload($project_id, 'TranslatableSource');
+        if ($confirmed != 1) {
+          $message = 'Not all the sources had been confirmed.';
+          $this->sendFileError('RestartPoint01', $message, NULL, $job_item);
+          throw new TMGMTException($message);
+        }
+
+        $job_item->getJob()->submitted('Job item has been successfully submitted for translation.');
```

Comment:
this is wrong. continuous jobs are not submitted. job items will be set to active instead soon. follow and help get the cron submission task in (step 10 I think), that will add inactive/active for job items.